Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountFolderNode.java

```diff
@@ -49,7 +49,7 @@ protected void refreshItems()
             throws AzureCmdException {
         if (clusterDetail != null && !clusterDetail.isEmulator()) {
             try {
-                clusterDetail.getConfigurationInfo(getProject());
```

Comment:
>getProject() [](start = 51, length = 12)

why remove this? How can we handle multi-instance of IDE?

Reply:
The passing `project` parameter is not used by any methods. And the project can be got from SubmitModel.

Reply:
Updated, as we studied, currently, mutli-instance IDE share a singleton Azure Subscription